doc/tutorial/Server.lhs

```diff
@@ -634,7 +634,7 @@ kind and abort early. The next two sections cover how to do just that.
 
 Other important instances from the list above are `MonadIO m => MonadIO
 (ExceptT e m)`, and therefore also `MonadIO Handler` as there is a `MonadIO IO` instance.
-[`MonadIO`](http://hackage.haskell.org/package/transformers-0.4.3.0/docs/Control-Monad-IO-Class.html)
+[`MonadIO`](http://hackage.haskell.org/package/base/docs/Control-Monad-IO-Class.html#t:MonadIO)
```

Comment:
`MonadIO` is in base since 4.9.0.0 which corresponds to GHC 8.0. 
Checking travis config in this repo I take it that you test with GHC 8+ only, so it makes sense to link to the current home of MonadIO.